crates/api_models/src/payments.rs

```diff
@@ -522,6 +522,8 @@ pub struct PaymentsRequest {
         example = "Custom_Order_id_123"
     )]
     pub merchant_order_reference_id: Option<String>,
+
+    pub skip_external_tax_calculation: Option<bool>,
```

Comment:
Add  appropriate  documentation above this field to be displayed in Api Reference docs

Reply:
done